dtvout

```diff
@@ -0,0 +1,2693 @@
+  CHKDT   Documentation/devicetree/bindings/extcon/wlf,arizona.yaml
```

Comment:
Please don't include this file, it should be generated when running the test.  Instead we should have some documentation about how to run it, and maybe a snippet from the output to show what it should look like.

Reply:
Sorry I meant the whole `dtvout` file - it needs to be generated during each test run.  But that's OK, we can keep this as an example and remove it when the test gets automated in KernelCI as we'll then be producing lots of test logs.